src/services/token.service.js

```diff
@@ -0,0 +1,40 @@
+import { Token } from '../models/token.js';
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
```

Comment:
Ensure that the `userId` field is correctly set up in the `Token` model to avoid potential issues with finding tokens by `userId`.